_posts/2016-03-28-chime-quality-assurance.markdown

```diff
@@ -0,0 +1,47 @@
+
+## Deciding what tools to use for Chime
+
+The newest Sittercity product, Chime, gave the QA team the opportunity to really work towards improving automation. There are many automation tools available. The foundational code a site uses will factor into what option is most appropriate. We primarily use ruby but also have a lot of different languages used in various projects. These may not work specifically for you, but they have met our needs so far: Watir, Cucumber, Appium, Applitools, and Jenkins.
+
+### Watir
+
+We chose to use [Watir](http://watir.com/) for the web automation. We have previously used it on our other [Sittercity](https://www.sittercity.com/) automation suites. It is easy to learn and very powerful in the right hands.
+
+### Cucumber
+Cucumber has many benefits for us. Since it's how we strive to structure our acceptance criteria. ♥ Cucumber and Watir-WebDriver sitting in a tree ♥ These two programs work really well with each other. It allows us to automate our web app and at the same time make the features easily readable to stakeholders, without any layers of abstraction.
+
+### Appium
+We have decided to try [Appium](appium.io) for our automated mobile testing. We use Calabash-iOS and Calabash-Android for our other mobile apps. However, we have ran into issues with them not being maintained for our needs in certain instances. This was why we decided to take a look at Appium. So far it's working well. We are looking forward to learning more about this tool and what opportunities it can provide for us.
+
+### Applitools
+
+Eyes_selenium is a gem by [Applitools](https://applitools.com/). This allows us to do automated UI testing. This allows us to test pixel and general layout level differences between pages. This is a great tool to use to make sure we don’t introduce any unpleasant visual bugs.
+
+### Jenkins
+
+We utilize Jenkins to its full extent to our QA team. It allows seamless continuous integration. Thanks to Jenkins we can automatically kick off automated regression suites for specific parts of our apps upon being deployed to an environment. QA is not required to start and monitor a build every time a deploy is done, which is awesome. The results of the suites are automatically messaged into chat rooms so everyone is aware when releases can continue to go out (or not). This allows QA to focus on more automation and do more exploratory testing of the app (which, in my opinion, is the end goal of all QA teams).
+
+## Automate all the things!
+
+By leveraging these tools together. Along with [parallelizing](https://github.com/grosser/parallel_tests) the tests themselves. We have helped the team gain greater confidence in making breaking changes as well as being able to deliver a lot in a short amount of time.
```

Comment:
I love this bulleted list but it's not formatted right. According to that markdown link it should look like this:

```
With all of these pieces combined we deliver the following to our company:

* We release to production multiple times a day.
* Run thousands of tests on a daily basis.
* Have drastically reduced regression suite running times.
* We are continuously improving/refactoring the automation suites.
* Make sure to leave time for ping pong
```

Make sure you put a newline between the previous sentence and the list. I wonder if that is why markdown is not picking up that you want a list.